lib/subprocessSupport.py

```diff
@@ -83,12 +101,29 @@ def iexe_cmd(cmd, useShell=False, stdin_data=None, child_env=None, text=True):
         # 3) Do not use communicate when you are dealing with multiple threads
         #    or processes at same time. It will serialize the process voiding
         #    any benefits from multiple processes
-        stdoutdata, stderrdata = process.communicate(input=stdin_data)
+        try:
+            stdoutdata, stderrdata = process.communicate(input=stdin_data, timeout=timeout)
+        except subprocess.TimeoutExpired as e:
+            process.kill()
+            stdoutdata, stderrdata = process.communicate()
+            err_str = "Timeout running '{}'\nStdout:{}\nStderr:{}\nException subprocess.TimeoutExpired:{}".format(
+                cmd,
+                stdoutdata,
+                stderrdata,
+                e,
+            )
```

Comment:
Could we use an fString (`f""`) here instead?

Reply:
This was done by the autoformatter to shorten the line (<120)